compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Serializability.hs

```diff
@@ -54,13 +55,13 @@ serializabilityConditionsType world0 _version mbModNameTpls vars = go
       TOptional typ -> go typ
       TMap typ -> go typ
       TNumeric (TNat n)
-          | n <= 38 -> noConditions
+          | n <= numericMaxScale -> noConditions
           | otherwise -> Left (URNumericOutOfRange n)
       TNumeric _ -> Left URNumericNotFixed
           -- We statically enforce bounds check for Numeric type,
-          -- requiring 0 <= n <= 38 for the argument to Numeric.
-          -- If the argument isn't given explicitly, we can't
-          -- guarantee serializability.
+          -- requiring 0 <= n <= 'numericMaxScale' for the argument
+          -- to Numeric. If the argument isn't given explicitly, we
+          -- can't guarantee serializability.
       TNat _ -> noConditions
```

Comment:
`TNat` should not be serializable at all. Most client languages can't express this. The only place where it's allowed in serializable types is immediately under `TNumeric`. I must have missed this in the previous PR.

Reply:
That makes sense. I'll update it, thanks!